src/views/catalog/utils/WizardVMContext/WizardVMContext.tsx

```diff
@@ -29,15 +60,26 @@ export const useWizardVM = (): WizardVMContextType => {
     }
   }, [vm]);
 
+  React.useEffect(() => {
+    // whenever the wizard metadata changes, save the data in session storage
+    if (wizardMetadata) {
```

Comment:
u need this check? i can see wizardMetadata is initialized, can it be null/undfiend?